rpc/types_transaction_receipt.go

```diff
@@ -25,6 +25,8 @@ const (
 type CommonTransactionReceipt struct {
 	// TransactionHash The hash identifying the transaction
 	TransactionHash *felt.Felt `json:"transaction_hash"`
+	BlockHash       *felt.Felt `json:"block_hash,omitempty"`
```

Comment:
Why did you add these fields here? They don't belong to the spec https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L2844

Reply:
Both the fields were being returned in the RPC response.
I did not cross-reference the specs. I'll revert this